### DISCUSSION TITLE
Blog RSS feed?

### DISCUSSION THREAD

**Discussion by Barugon** [FyroxEngine/Fyrox#397] [Q&A]
Is there a RSS feed for the Fyrox blog?

**Comment by Barugon** [comment]
I guess not.

**Comment by mrDIMAS** [comment]
There's no RSS feed yet, but there's an issue about it - https://github.com/FyroxEngine/rg3d.rs/issues/35.

**Comment by mrDIMAS** [comment]
It should work now.
